src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023_convert_session.py

```diff
@@ -52,6 +52,7 @@ def session_to_nwb(
         holographic_stimulation_file_path=holographic_stimulation_file_path,
         epoch_name=epoch_name,
         verbose=False,
+        stub_test=stub_test,
     )
 
     conversion_options = {
```

Comment:
I believe I have found your bug. Based on the `baseimagingextractorinterface` from neuroconv, `stub_test` needs to be passed in as a conversion option to the `add_to_nwb` method. It is specified accordingly in line 59. But, then in line 64, `conversion_options` is overwritten, so `stub_test` is not propagated properly.

Reply:
Thank man! I totally missed that! 
It's working now